package-lock.json

```diff
@@ -12784,8 +12784,7 @@
           "version": "3.0.0",
           "resolved": "https://registry.npmjs.org/get-stream/-/get-stream-3.0.0.tgz",
           "integrity": "sha1-jpQ9E1jcN1VQVOy+LtsFqhdO3hQ=",
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
I'm not sure if all these lock file changes should be here?

Reply:
No they shouldn't

Reply:
This file should not be commited

Reply:
Reverted it.